xendit-java-lib/src/main/java/com/xendit/model/EWalletPayment.java

```diff
@@ -136,7 +136,20 @@ public static EWalletPayment createDanaPayment(
     params.put("callback_url", callbackUrl);
     params.put("redirect_url", redirectUrl);
     params.put("ewallet_type", EWalletType.DANA);
-    return createPaymentRequest(params);
+    return createPaymentRequest(new HashMap<>(), params);
+  }
+
+  /**
+   * Create new ewallet payment with all parameters as HashMap
+   *
+   * @param headers
+   * @param params mount end-customer will pay.
+   * @return EWalletPayment
+   * @throws XenditException XenditException
+   */
+  public static EWalletPayment createEWalletPayment(
```

Comment:
Can this be renamed as `createPaymentRequest`?

Reply:
How about `createPayment` ? Payment request sounds like someone needs to approve the payment request from the other side 😁

Reply:
No it's meant to be an override of the existing method. I think consistency is more important

Reply:
Also paymentRequest is the right term